### PR TITLE
build: Update Swagger to 0.30.3

### PR DESCRIPTION
**Pull request by jrajahalme**
Update Swagger to 0.30.x that is the first version with arm64 compatible images at quay.io/goswagger/swagger. This allows Cilium APIs to be regenerated on arm64 platforms.

Signed-off-by: Jarno Rajahalme <jarno@isovalent.com>

### CODE REVIEW

**Review by gandro — APPROVED** [review]
Scrolled a bit through the generated code for my code owners, seems like not much has changed. Looks good to me!

**Comment by jrajahalme** [comment]
@gandro Had to push a fixed version as endpoint state in Endpoint Status had changed to a pointer, which required changes in couple of call cites. Not sure if we could change the input yaml instead to keep the old non-pointer.

**Comment by aanm** [comment]
@jrajahalme it seems the CI has legitimate failures.

**Comment by aanm** [comment]
Marking the PR as draft since CI is failing. Feel free to mark it as ready for review once CI is fixed.

**Comment by jrajahalme** [comment]
@aanm Status pointer comparisons were not a good idea, fixed (`cilium policy wait` was broken because of this). Hopefully CI passes now :-)

**Review by aanm — APPROVED** [review]

File: api/v1/models/endpoint_state.go [hunk 1]
```diff
@@ -21,16 +22,25 @@ import (
...
+func (m EndpointState) Pointer() *EndpointState {
+	return &m
+}
+
 const (
 
-	// EndpointStateWaitingForIdentity captures enum value "waiting-for-identity"
-	EndpointStateWaitingForIdentity EndpointState = "waiting-for-identity"
+	// EndpointStateWaitingDashForDashIdentity captures enum value "waiting-for-identity"
+	EndpointStateWaitingDashForDashIdentity EndpointState = "waiting-for-identity"
```
Comment (christarazi) [inline api/v1/models/endpoint_state.go:37]: This seems not desirable. Is there a way to avoid this?

**Review by tklauser — APPROVED** [review]

File: api/v1/models/endpoint_state.go (hunk 1 above)
Comment (jrajahalme) [reply api/v1/models/endpoint_state.go:37]: @christarazi Does not look like there is an option to get around this: https://github.com/go-swagger/go-swagger/issues/2515

**Comment by jrajahalme** [comment]
Rebased.

**Review by christarazi — APPROVED** [review]

**Pull request merged**